### PR TITLE
Fixes for duplicated othernodes, date/time logging for crashcloudwatch and buffer_size for init-quorum

### PR DESCRIPTION
**Pull request by ExcChua**
Added datetime to crashcloudwatch and exitcodes to quorum, so that crashquorum doesn't emit a cloudwatch when quorum is stopped by supervisorctl.

### CODE REVIEW

**Comment by ExcChua** [comment]
Prevent duplicated othernodes in init-bootnodes.sh
Updated init-quorum.sh with buffer_size
Updated crashcloudwatch.py to log with date/time.

File: packer/instance-scripts/crashcloudwatch.py [hunk 1]
```diff
@@ -102,7 +103,7 @@ def runforever(self, test=False):
                 # do nothing with non-TICK events
                 childutils.listener.ok(self.stdout)
                 if test:
-                    self.stderr.write('non-exited event\n')
+                    self.stderr.write(datetime.now().strftime("%Y-%m-%d_%H:%M:%S") + ' non-exited event\n')
```
Comment (Lsquared13) [inline packer/instance-scripts/crashcloudwatch.py]: nit: Throw the date template string into a global variable at the top

File: packer/instance-scripts/init-bootnode.sh [hunk 2]
```diff
@@ -20,25 +20,32 @@ function complete_constellation_config {
 
     local REGIONS=$(cat /opt/quorum/info/regions.txt)
 
-    # Configure constellation with other node IPs
-    OTHER_NODES=""
-    for region in ${REGIONS[@]}
-    do
-        local NUM_BOOTNODES=$(cat /opt/quorum/info/bootnode-counts/${region}.txt)
-        for index in $(seq 0 $(expr $NUM_BOOTNODES - 1))
+    DIR=$(dirname "${CONSTELLATION_CONFIG_PATH}")
```
Comment (Lsquared13) [inline packer/instance-scripts/init-bootnode.sh]: Make these variables local please. If you do need access to it outside the function, create the variable in the main code path and pass it to the function as an argument.

**Review by Lsquared13 — CHANGES_REQUESTED** [review]
Please confirm that you've run the private transaction test on a small single region network

File: packer/instance-scripts/init-bootnode.sh (hunk 2 above)
Comment (ExcChua) [reply packer/instance-scripts/init-bootnode.sh]: After making these changes, do I do a new PR, or what? No idea how to proceed.

**Comment by Lsquared13** [comment]
After addressing the changes, I think you just push to the source branch `EximChua:master` and it updates automatically (for that reason I recommend you don't make your PRs from master, just to make your life easier if you want to do more work on the repo).

If that doesn't work, ask some other folks who have actually done PRs regularly, but I think that's how it goes

File: packer/instance-scripts/init-bootnode.sh (hunk 2 above)
Comment (Lsquared13) [reply packer/instance-scripts/init-bootnode.sh]: See main thread comment below.

**Comment by ExcChua** [comment]
transaction test was ran on the network, and it worked.

**Review by Lsquared13 — APPROVED** [review]

**Comment by ExcChua** [comment]
Merge completed.

**Pull request merged**